src/frontend/nginx.conf

```diff
@@ -12,7 +12,7 @@ server {
         add_header X-Content-Type-Options "nosniff" always;
         add_header X-XSS-Protection "1; mode=block";
         add_header Pragma "no-cache";
-        add_header Content-Security-Policy "default-src 'self';";
+        add_header Content-Security-Policy "default-src 'self' https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/js/bootstrap.bundle.min.js;";
```

Comment:
Intentionally refraining from adding the wildcard / complete domain [mentioned in prev comment as well]. Are we ok to allow anything from cdn.jsdelivr.net? I thought it's safe that we allow only the two needed subpaths for src and style - and nothing else.

Reply:
> also you need to add one for css (style-src) and js (script-src)

Already added these two; But can add like your syntax - makes it more explicit 👍. However still makes it further secure to keep the complete path instead of the whole domain.

Reply:
I won't add explicit paths. It can easily break. The domain should suffice. I won't consider a domain https://cdn.jsdelivr.net/; a wildcard.  It would be strange if a scanner complains it needs explicit paths. It will make the application fragile. 

My 2 cents.